cmd/namespace.go

```diff
@@ -97,81 +93,96 @@ func listNamespaceCmd() *cobra.Command {
 }
 
 func createNamespaceCmd() *cobra.Command {
-	var host, format, comp string
-	var desc string
+	var host, id, desc, format, comp string
 	var req stencilv1beta1.CreateNamespaceRequest
 
 	cmd := &cobra.Command{
 		Use:   "create",
 		Short: "Create a namespace",
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.ExactArgs(0),
 		Example: heredoc.Doc(`
-			$ stencil namespace create <namespace-id> --format=<schema-format> --comp=<schema-compatibility> --desc=<description> 
+			$ stencil namespace create 
+			$ stencil namespace create -n=odpf -f=FORMAT_PROTOBUF --c=COMPATIBILITY_BACKWARD -d="Event schemas"
```

Comment:
should `--c` be a single dash?
```suggestion
			$ stencil namespace create -n=odpf -f=FORMAT_PROTOBUF -c=COMPATIBILITY_BACKWARD -d="Event schemas"
```

Reply:
Yes, Fixed.